.github/workflows/deploy_bookdown.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  push:
+     branches:
+       - master
+
+  
+
+name: renderbook
+
+jobs:
+  bookdown:
+    name: Render-Book
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: r-lib/actions/setup-r@v1
+      - uses: r-lib/actions/setup-pandoc@v1
+      - name: Install rmarkdown
+        run: Rscript -e 'install.packages(c("rmarkdown","bookdown", "car", "afex", "psych", "factoextra"))'
+      - name: Render Book
+        run: Rscript -e 'bookdown::render_book("index.Rmd")'
+      - uses: actions/upload-artifact@v1
+        with:
+          name: _book
+          path: _book/
+  
+# Need to first create an empty gh-pages branch
+# see https://pkgdown.r-lib.org/reference/deploy_site_github.html
+# and also add secrets for a GH_PAT and EMAIL to the repository
+# gh-action from Cecilapp/GitHub-Pages-deploy
+  checkout-and-deploy:
+   runs-on: ubuntu-latest
+   needs: bookdown
+   steps:
+     - name: Checkout
+       uses: actions/checkout@master
+     - name: Download artifact
+       uses: actions/download-artifact@v1.0.0
+       with:
+         # Artifact name
+         name: _book # optional
+         # Destination path
+         path: _book # optional
+     - name: Deploy to GitHub Pages
+       uses: Cecilapp/GitHub-Pages-deploy@master
```

Comment:
Did we verify that this action `Cecilapp/GitHub-Pages-deploy` doesn't do anything shady?

Also, `@master` may break at any point. Aren't there any releases we can tag? For example, `actions/download-artifact@v1.0.0` is a release so it will always stay the same even if `actions/download-artifact@master` is updated.

Reply:
Don recommends

       `uses: Cecilapp/GitHub-Pages-deploy@v3.2.1`